### ISSUE TITLE
Ember Inspector installed in temporary location

### ISSUE THREAD

**Issue by jacobq** [felixrieseberg/ember-electron#224]
For some reason, it seems that an old temp path (on generated on last `ember electron` run) is used to load the Ember Inspector. I think what is happening is that after we install it (via `addDevToolsExtension`) electron is remembering the path we supplied, which is a temporary one and changes from one execution to another.

Indeed, looking at `~/.config/Electron/DevTools\ Extensions` (path may vary by platform -- see `app.getPath('userData')`) shows that `devtron` is located in the `node_modules` folder at the root of an ember-electron project, but `ember-inspector` is in a `node_modules` folder inside of a `tmp/electron_livereload-XXX.tmp` folder.

Neither of these is truly "right" since this folder is shared by multiple projects, but I think we should at least attempt to use a non-temp location to eliminate the error message. If possible, it'd be nice to prefer a globally installed version too, if one is available.

## Steps to reproduce

* Platform: Linux x64
* `node` v6.10.2
* `npm` v3.10.10
* `ember-electron` v2.0.1
* `electron-prebuilt-compile` v1.6.2

```
ember new junk
cd junk
ember install ember-electron
npm install
bower install
ember electron
```

Running the below script in the devTools console uninstalls the Ember Inspector, which appears to temporarily alleviates the problem (doesn't show up on next build, but comes back after that):
```
var bw = requireNode('electron').remote.BrowserWindow;
var ei = bw.getDevToolsExtensions()['Ember Inspector'];
if (ei) {
    bw.removeDevToolsExtension('Ember Inspector');
}
```

## Sample output:

```
$ ember electron

"ember-welcome-page" was detected in your devDependencies!

Please note that this addon only works in development environment
and will not render in production mode. It is safe to uninstall
this addon once you removed the {{welcome-page}} template tag.


ember-electron/.eslintrc.js: line 1, col 1, 'module' is not defined.

1 error

===== 1 JSHint Error

New output path: /path/to/junk/tmp/electron_livereload-GvsxmhuQ.tmp

Build successful - 5156ms.



Slowest Nodes (totalTime => 5% )              | Total (avg)         
----------------------------------------------+---------------------
Babel (16)                                    | 4832ms (302 ms)     

Starting Electron...
Reading /last/run/project/tmp/electron_livereload-hZQ82TnP.tmp/node_modules/ember-inspector/dist/chrome/manifest.json failed.
Error: ENOENT: no such file or directory, open '/last/run/project/tmp/electron_livereload-hZQ82TnP.tmp/node_modules/ember-inspector/dist/chrome/manifest.json'
    at Object.fs.openSync (fs.js:558:18)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:173:20)
    at Object.fs.readFileSync (fs.js:468:33)
    at Object.fs.readFileSync (ELECTRON_ASAR.js:506:29)
    at getManifestFromPath (/path/to/junk/node_modules/electron-prebuilt-compile/node_modules/electron/dist/resources/electron.asar/browser/chrome-extension.js:34:26)
    at App.<anonymous> (/path/to/junk/node_modules/electron-prebuilt-compile/node_modules/electron/dist/resources/electron.asar/browser/chrome-extension.js:363:26)
    at Object.onceWrapper (events.js:290:19)
    at emitTwo (events.js:111:20)
    at App.emit (events.js:191:7)
```

See also #104 & #30

**Comment by xn** [comment]
Stung by this as well.

**Comment by jacobq** [comment]
@xn, a work-around for this is to edit the  `DevTools Extensions` JSON (array) file (in the electron userData path, e.g. `~/.config/Electron`) and replace the temporary path with a stable one (e.g. global installation).

**Comment by xn** [comment]
Thanks

**Comment by pichfl** [comment]
@jacobq can you check if this was resolved with 2.1.1?

**Comment by jacobq** [comment]
@pichfl **Confirmed**; it appears to work fine in v2.1.1.
